pkg/globalnet/controllers/iptables/iface.go

```diff
@@ -18,19 +18,24 @@ limitations under the License.
 package iptables
 
 import (
+	"crypto/sha256"
+	"encoding/base32"
 	"fmt"
 	"strings"
 
 	"github.com/submariner-io/admiral/pkg/log"
 	"github.com/submariner-io/submariner/pkg/iptables"
 	"github.com/submariner-io/submariner/pkg/routeagent_driver/constants"
+	corev1 "k8s.io/api/core/v1"
 
 	"k8s.io/klog"
 )
 
 type Interface interface {
 	AddClusterEgressRules(sourceIP, snatIP, globalNetIPTableMark string) error
 	RemoveClusterEgressRules(sourceIP, snatIP, globalNetIPTableMark string) error
+	UpdateIngressRulesForService(globalIP, chainName string, addRules bool) error
```

Comment:
Instead of passing a bool, how about defining Add/Remove functions as we above:

```suggestion
        AddIngressRulesForService(globalIP, chainName string) error
        RemoveIngressRulesForService(globalIP, chainName string) error
```